main.py

```diff
@@ -27,8 +27,8 @@ async def on_message(message):
     if message.author == client.user:
         return
 
-    if message.content.startswith('$hello'):
-        await message.channel.send('Fuck you!')
+    if message.content.startswith("$hello"):
+        await message.channel.send("Fuck you!")
```

Comment:
What is the reason to use double quotes?

Reply:
black
I assume it's adjustable. Maybe it has to do partially with the fact that in other languages (c, c++) double quotes are strings , and single quotes are characters.